src/components/BulkEnrollmentPage/stepper/ReviewStep.jsx

```diff
@@ -7,9 +7,9 @@ import { ADD_LEARNERS_STEP, ADD_COURSES_STEP, REVIEW_TITLE } from './constants';
 import ReviewList from './ReviewList';
 
 const EMAILS = {
```

Comment:
Heh, let's change the name of the constant as well.

Reply:
hahaha good point